Cargo.toml

```diff
@@ -18,7 +18,8 @@ enable-chaos-mode-by-default = ["ckb-vm-definitions/enable-chaos-mode-by-default
 [dependencies]
 byteorder = "1"
 bytes = "0.5.4"
-goblin = "0.2.0"
+goblin_v0 = { package = "goblin", version = "0.2.0" }
```

Comment:
I think it might be better if we stick to goblin's own versioning here, such as `goblin_v020` and `goblin_v034`. CKB VM has the concept of v0 and v1, but goblin does not.

Reply:
Agree with this view.